frontend/packages/console-shared/src/components/formik-fields/text-column-field/TextColumnField.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { DndProvider } from 'react-dnd';
+import HTML5Backend from 'react-dnd-html5-backend';
+import { FieldArray, useField } from 'formik';
+import { FormGroup } from '@patternfly/react-core';
+import { useFormikValidationFix } from '@console/shared';
+import MultiColumnFieldFooter from '../multi-column-field/MultiColumnFieldFooter';
+import { getFieldId } from '../field-utils';
+import TextColumnItem from './TextColumnItem';
+import { TextColumnFieldProps } from './text-column-types';
+
+const TextColumnField: React.FC<TextColumnFieldProps> = (props) => {
+  const { required, name, label, addLabel, helpText, isReadOnly, onChange } = props;
+  const [field, { touched, error }] = useField<string[]>(name);
+  useFormikValidationFix(field.value);
+  const rowValues = field.value ?? [];
+  const fieldId = getFieldId(name, 'single-column');
+  const isValid = !(touched && error);
+  return (
+    <FieldArray
+      name={name}
+      render={(arrayHelpers) => {
+        return (
+          <>
+            <DndProvider backend={HTML5Backend}>
+              <FormGroup
+                fieldId={fieldId}
+                label={label}
+                validated={isValid ? 'default' : 'error'}
+                isRequired={required}
+                helperText={helpText}
+              >
+                {rowValues.map((v, idx) => {
+                  return (
+                    <TextColumnItem
+                      key={`${idx.toString()}`}
+                      idx={idx}
+                      arrayHelpers={arrayHelpers}
+                      rowValues={rowValues}
+                      {...props}
+                    />
+                  );
+                })}
+              </FormGroup>
+            </DndProvider>
+            {!isReadOnly && (
+              <MultiColumnFieldFooter
+                addLabel={addLabel}
+                onAdd={() => {
+                  const newValue = '';
+                  arrayHelpers.push(newValue);
```

Comment:
```suggestion
                  arrayHelpers.push('');
```

Reply:
Ah, this is from old code I just did refactor but will get this updated.